### DISCUSSION TITLE
How to activate italian locale after installing this package?

### DISCUSSION THREAD

**Discussion by realtebo** [Laravel-Lang/common#27] [Q&A] [answered]
I just discovered and installed your package.

I already have in config/app.php the locale set to 'it'.

But ... now ..?

How to publish 'it' folder under resources\lang ? I still have only 'en'

**Comment by mohamedsabil83** [comment, marked as answer]
Just install [publisher](https://github.com/Laravel-Lang/publisher) and publish any locale you want.
For your request, after install **publisher**, run `php artisan lang:add it`

**Reply by realtebo** [reply]
Doing this, I got a new 'it' folder but files into the folder are all empty !

**Reply by realtebo** [reply]
I created an issue on publisher repo: https://github.com/Laravel-Lang/publisher/issues/200

**Reply by mohamedsabil83** [reply]
weird, just try that and all **it** files published.

which version do you have used?

**Reply by realtebo** [reply]
Lang Version: 8.1.3
Lang Publisher Version: 11.1.5
Laravel Version: 8.79.0
PHP Version: 7.4

**Reply by realtebo** [reply]
I am trying removing vendor and reinstalling all..

**Reply by mohamedsabil83** [reply]
just install **laravel-lang/publisher**. it would install and use **laravel-lang/lang** under the hood.

**Reply by realtebo** [reply]
Yes, it worked, but it is mandatory to BEFORE remove laravel-lang/lang from composer.

Also, please note that setup instructions tell to install v8 and it's a lot outdated. 
Also, instructions should bring user to use the publisher instead to use it directly.
Or, at least, it should be explained
- you can install and copy it locales
- or instead install the publisher and use the console command

I thank you, and suggest a little editing of install infos

**Reply by mohamedsabil83** [reply]
This package aims to translate all Laravel core phrases. For help managing and transferring needed locales to dev project, there are many packages for that which some of them mentioned in the **README** under the title [Translation managers](https://github.com/Laravel-Lang/lang#translation-managers)

**Reply by realtebo** [reply]
For future googlers. If you use publisher pachakge, remove this package from dev-dep because the publisher uses it automatically

**Reply by andrey-helldar** [reply]
@realtebo, we fixed package compatibility: https://github.com/Laravel-Lang/publisher/issues/204
